src/Aggregation.ts

```diff
@@ -0,0 +1,54 @@
+/** AggreationType will be used to create type of aggreation when querying history states.
+  <ul>
+  <li>AggregationType.COUNT: count function.</li>
+  <li>AggregationType.SUM: sum function. </li>
+  <li>AggregationType.MAX: max function.</li>
+  <li>AggregationType.MIN: min function.</li>
+  <li>AggregationType.MEAN: mean function.</li>
+  </ul>
+*/
+export const FunctionType = {
+    COUNT: "COUNT",
+    SUM: "SUM",
```

Comment:
`FunctionType` follows design of other similar ones, like `Site`, `Types`, and etc. 
Let me revaluate again. If necessary, I will change all the enum/const variables.